### ISSUE TITLE
Query - basic widgets

### ISSUE THREAD

**Issue by richardbeare** [IRkernel/IRkernel#430]
Hi,
Apologies if this is off topic. I'm struggling to find information about this. Is there any equivalent of ipywidgets allowing basic feedback to R? I'm aware of the likes of plotly and htmlwidgets, but have not been able to identify any mechanism for providing data back to R. There are some comments around the release of ipywidgets 5.0 claiming that recent refactoring is designed to make writing such tools more feasible for kernel authors - is there any work on these fronts that you are aware of?

Thaks

**Comment by flying-sheep** [comment]
yeah, the new declarative widgets work with it: http://jupyter-incubator.github.io/declarativewidgets/

check the readme [here](https://github.com/jupyter-incubator/declarativewidgets), it tells you about `installr`.

there’s example notebooks for r and other languages in [here](https://github.com/jupyter-incubator/declarativewidgets/tree/master/etc/notebooks/examples)
